tests/www/apply/test_process.py

```diff
@@ -258,6 +271,128 @@ def test_details_for_unauthorized_prescriber(self, *args, **kwargs):
         self.assertNotContains(response, "Supersecretname")
         self.assertNotContains(response, "Unknown")
 
+    def test_details_for_job_seeker(self, *args, **kwargs):
+        """As a job seeker, I can access the job_applications details for job seekers."""
+        job_seeker = JobSeekerFactory()
+
+        job_application = JobApplicationFactory(job_seeker=job_seeker, job_seeker_with_address=True)
+        job_application.process()
+
+        self.client.force_login(job_seeker)
+
+        url = reverse("apply:details_for_jobseeker", kwargs={"job_application_id": job_application.pk})
+        response = self.client.get(url)
+        self.assertContains(response, format_nir(job_seeker.nir))
+        self.assertContains(response, job_seeker.email)
+        self.assertContains(response, job_seeker.post_code)
+        self.assertContains(response, job_seeker.address_line_1)
+        self.assertContains(response, job_seeker.city)
+        self.assertContains(response, f"Prénom : <b>{job_seeker.first_name}</b>", html=True)
+        self.assertContains(response, f"Nom : <b>{job_seeker.last_name}</b>", html=True)
+        self.assertContains(
+            response,
+            f'<span class="text-muted">{job_seeker.first_name} {job_seeker.last_name.upper()}</span>',
+            html=True,
+        )
+
+        # phone sender is hidden for job seeker
+        self.assertNotContains(response, format_phone(job_application.sender.phone))
+
+    def test_details_for_job_seeker_as_other_user(self, *args, **kwargs):
+        job_application = JobApplicationFactory()
+        url = reverse("apply:details_for_jobseeker", kwargs={"job_application_id": job_application.pk})
+
+        for user in [
+            JobSeekerFactory(),
+            EmployerFactory(with_siae=True),
+            PrescriberFactory(),
+            LaborInspectorFactory(membership=True),
+        ]:
+            with self.subTest(user=user):
+                self.client.force_login(user)
+                response = self.client.get(url)
+                assert response.status_code == 404
+
+    def test_details_for_prescriber_with_transition_logs(self, *args, **kwargs):
+        """As a prescriber, I can access transition logs for job_applications details for prescribers."""
+        job_application = JobApplicationFactory(sent_by_authorized_prescriber_organisation=True)
+
+        # transition logs setup
+        job_application.process()
+        jatl = job_application.logs.first()
+        jatl.user = job_application.to_siae.active_members.first()
+        jatl.save()
```

Comment:
```suggestion
        job_application.process(user=job_application.to_siae.active_members.first())
```
devrait faire ce que tu veux je pense.